src/Generators/Microsoft.Gen.ComplianceReports/Emitter.cs

```diff
@@ -21,11 +21,15 @@ public Emitter()
     }
 
     [SuppressMessage("Performance", "LA0002:Use 'Microsoft.Extensions.Text.NumericExtensions.ToInvariantString' for improved performance", Justification = "Can't use that in a generator")]
-    public string Emit(IReadOnlyCollection<ClassifiedType> classifiedTypes, string assemblyName)
+    public string Emit(IReadOnlyCollection<ClassifiedType> classifiedTypes, string assemblyName, bool includeName = true) // show or hide assemblyName in the report,defaulted to true.
```

Comment:
This is added because you have a new reported dependent on this , and Compliance report is only a part of that bigger report, which have the assemblyName at the very top of it. 

so we dont need to replicate the assemblyName in the properties. 

The value is defaulted to true, to avoid any breaking changes for backward computability